chembl_structure_pipeline/standardizer.py

```diff
@@ -470,21 +470,30 @@ def reapply_molblock_wedging(m):
                 b.SetBondDir(Chem.BondDir.BEGINDASH)
 
 
+def parse_molblock(ctab, useRDKitChemistry=False):
+    if useRDKitChemistry:
+        m = Chem.MolFromMolBlock(ctab, sanitize=False, removeHs=True)
```

Comment:
Yep, I did it because I thought that I preferred it to throw an exception instead of getting a None but after writing the tests I'll keep it as you say. 
I'm keeping explicit arguments to be sure that the behaviour doesn't change if RDKit default values change.